paginator/paginator.go

```diff
@@ -130,6 +131,10 @@ func (p *Paginator) setup(db *gorm.DB, dest interface{}) {
 			sqlKey := p.parseSQLKey(dest, p.rules[i].Key)
 			p.rules[i].SQLRepr = fmt.Sprintf("%s.%s", sqlTable, sqlKey)
 		}
+		if p.rules[i].ReplaceNULLWith != nil {
+			data, _ := json.Marshal(p.rules[i].ReplaceNULLWith)
+			p.rules[i].SQLRepr = fmt.Sprintf("COALESCE(%s, '%v')", p.rules[i].SQLRepr, string(data))
```

Comment:
Not sure this is the best approach to transform Go objects into string representation for SQL. Any other ideas?